### ISSUE TITLE
Bug - Not able to enter decimals 

### ISSUE THREAD

**Issue by MaudSimon** [alephium/alephium-frontend#151]
**Describe the bug**
Mobile Wallet, IOS Iphone 10. In some IOS versions or most like certain language set-up,  the keyboard to input the amount when sending assets is a "," and not a "." as a result it is not possible to specify a non-integer number of coins to send. When you click on the comma, the input field is cleared. Could this be because the keyboard suggests a comma instead of a period? Attached a video with the bug.

**To Reproduce**
I'm assuming this is language specific, so one would need to set his iphone in the language in question (or any language that results the keyboard to show "," instead of "." 
Asking the user for his language set-up and IOS version.

**Expected behavior**
User expected to be able to enter decimals.

**Screenshots**

Uploading RPReplay_Final1701682546.mp4…

**Environment**

- OS: IOS 
- Iphone 10
- Version [e.g. 1.0.1]

**Comment by MaudSimon** [comment]
User wallet is in Russian.

**Comment by nop33** [comment]
Yes, this is because we are expecting a "." but receiving a ",". Maybe the wallet could automatically substitute commas for fullstops. Thanks for this report.
